test/test_quiz_mode.py

```diff
@@ -117,6 +117,7 @@ def test_right_answer_count_update(self):
         new_task_response = self.app.get(new_task_url)
         task = json.loads(new_task_response.data)
         task_run_url = '/api/taskrun'
+        print(task_answers, task)
```

Comment:
Just highlighting a `print` statement, in case of clean-up. Thanks.

Reply:
Good catch.  Removing.